repo/packages/C/cassandra/0/marathon.json.mustache

```diff
@@ -8,8 +8,8 @@
     0
   ],
   "uris": [
-    "{{resource.assets.uris.cassandra-mesos-0-2-0-1-tar-gz}}",
-    "{{resource.assets.uris.jre-7u76-linux-x64-tar-gz}}"
+    "{{cassandra.binary}}",
```

Comment:
Why are we not grabbing it from the `resource.json`? This looks like it is going to break our packaging solution for hosting on premise and upgrade story.